src/components/LmsConfigurations/index.test.jsx

```diff
@@ -139,6 +159,19 @@ describe('<LmsConfigurations /> ', () => {
     expect(wrapper.find('SuccessFactorsIntegrationConfigForm').props().config).toEqual(wrapper.state().sapsfConfig);
   });
 
+  it('get the Degreed LMS configuration when present', async () => {
```

Comment:
nitiest of nits: `gets` instead of `get` so it can be read as a sentence: "It gets the..."

Reply:
FWIW, seems like all the methods use `get`, so my only ask is that we stay consistent!